site/content/docs/5.0/guidelines/global-headers.md

```diff
@@ -7,6 +7,633 @@ subgroup: Modules
 toc: true
 ---
 
-{{< callout info >}}
+<!-- {{< callout info >}}
 The feature will be delivered with [#525]({{< param repo >}}/issues/525).
-{{< /callout >}}
+{{< /callout >}} -->
+
+<main>
+  <div class="container">
+    <h2 id="suprabar">Supra bar</h2>
+    <p>
+      <strong>Supra bars are hidden on tablet and mobile devices.</strong>
+      Based on viewport size, supra bar is hidden under <var>768px</var>.
+    </p>
+    <h3 class="mt-5 h5">Supra bar standard — <a id="web-glh-sub-001" class="ui-kit-id">web-glh-sub-001</a></h3>
+  </div>
+  <nav class="navbar navbar-dark bg-dark navbar-expand-md supra">
```

Comment:
I'm not saying that the following code must replace yours here; I just tried a quick implementation and played with the `.navbar*` classes. But maybe you would be able to develop this example without this `supra` class:

```html
<nav role="navigation" class="navbar navbar-dark bg-dark navbar-expand-md pt-0">
  <div class="container">
    <div class="navbar-nav">
      <a href="#" class="nav-link active pb-1" aria-current="page">Personal</a>
      <a class="nav-link pb-1" href="#">Business</a>
    </div>
  </div>
</nav>
```

Reply:
It seems good enough indeed, I might base myself on this, thanks !